Changelog.md

```diff
@@ -4,6 +4,10 @@ All notable changes will be documented in this file.
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [Unreleased]
```

Comment:
I forgot that we have a changelog for this repository, as well. What do you think, should we remove it, and only rely on "github releases"?